### PR TITLE
[FIX] fix web form parent template

### PR DESCRIPTION
**Pull request by victor-champonnois**
Related to this [PR](https://github.com/coopiteasy/vertical-cooperative/pull/200) and this [task ](https://gestion.coopiteasy.be/web#id=2349&view_type=form&model=project.task&action=479). 

- fix the template inheritance
- adapt the form design to be consistent with the parent form

### CODE REVIEW

**Comment by victor-champonnois** [comment]
@robinkeunen @carmenbianca I need a review (and an approval :smile: ) in order to merge it. This is needed in order for PRs in this repo to pass the CI (because this is a companion PR to the EMC form vertical-coop PR that has already been merged)

File: beesdoo_easy_my_coop/views/subscription_templates.xml [hunk 1]
```diff
@@ -4,35 +4,30 @@
...
+                        name="info_session_confirmed"
+                        class="form-check-input"
+                        t-att-required="info_session_required"
+                        t-att-checked="info_session_confirmed"
+                    />
+                    <label>
                             <t t-raw="info_session_text" />
-                        </div>
-                    </div>
+                    </label>
```
Comment (carmenbianca) [inline beesdoo_easy_my_coop/views/subscription_templates.xml]: Bad indentation

**Review by carmenbianca — APPROVED** [review]

File: beesdoo_easy_my_coop/views/subscription_templates.xml (hunk 1 above)
Comment (victor-champonnois) [reply beesdoo_easy_my_coop/views/subscription_templates.xml]: fixed, thanks :)

Comment (carmenbianca) [reply beesdoo_easy_my_coop/views/subscription_templates.xml]: I don't think you successfully fixed it :x

Comment (victor-champonnois) [reply beesdoo_easy_my_coop/views/subscription_templates.xml]: Ah, maybe we didn't look at the same line, which one is badly indented ?

Comment (victor-champonnois) [reply beesdoo_easy_my_coop/views/subscription_templates.xml]: ah ignore my comment

Comment (victor-champonnois) [reply beesdoo_easy_my_coop/views/subscription_templates.xml]: Fixed (I hope :smile: ), I used a reformat tool

Comment (carmenbianca) [reply beesdoo_easy_my_coop/views/subscription_templates.xml]: :tada: It's fixed!

**Review by carmenbianca — APPROVED** [review]

**Pull request merged**